pkgs/data/fonts/ia-writer-quattro/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenvNoCC
+, fetchFromGitHub
+, lib
+,
+}:
+stdenvNoCC.mkDerivation rec {
+  pname = "ia-writer-quattro";
+  version = "f32c04c3058a75d7ce28919ce70fe8800817491b";
```

Comment:
`version` should be `unstable-YYYY-MM-DD`, with the date of the commit that is used.  See the [pkgs readme](https://github.com/NixOS/nixpkgs/tree/master/pkgs#package-naming) for more information.

You should also use that version in your init commit message: `<pname>: init at <version>`

Reply:
Hi, thanks for the suggestions! I'm relatively new to contributing to open source, so bear with me :D. Should I open a new pull request with your suggestions?

Reply:
I think I've fixed everything. Please let me know if there's anything else!